src/pfe/portal/routes/projects/metrics.route.js

```diff
@@ -187,6 +190,10 @@ router.post('/api/v1/projects/:id/metrics/inject', validateReq, async function (
       injectMetrics: injectMetrics
     });
 
+    await metricsService.injectMetricsCollectorIntoProject(project.projectType, path.join(project.workspace, project.directory));
+    // We now need to synch files in to the build container
+    copyProjectToBuild(project);
```

Comment:
if we don't `await` this (and `user.buildProject(...)` below), then our try-catch block won't catch any errors from them. To achieve Howard's  requested change, maybe we should keep this `await`, add an `await` onto `buildProject`, then move `res.sendStatus(202)` above them?

I'd be tempted to add a response body informing the user why they're getting a 202 rather than 200 (e.g. `res.status(202).send('Metrics collector successfully injected. Project rebuild queued.');`